scripts/deploy.sh

```diff
@@ -18,7 +18,7 @@ declare branch cluster_size package_version docker_image
 
 : ${HOPRD_API_TOKEN:?"env var missing"}
 : ${HOPRD_PASSWORD:?"env var missing"}
-: ${FUNDING_PRIV_KEY:?"env var missing"}
+: ${FAUCET_SECRET_API_KEY:?"env var missing"}
```

Comment:
Should the script actually _use_ the environment variable anywhere?

Reply:
No. This is to make sure that the var is set because it is needed later in a sub-script.